### DISCUSSION TITLE
Popup inside input field

### DISCUSSION THREAD

**Discussion by akashramtohul4** [quasarframework/quasar#11478] [General - Components / Directives / etc]
Hello friends,

Am try to build a popup inside a input field which inside the popup will contain some information .Below in an image of how i want it to look like
![post](https://user-images.githubusercontent.com/84211129/143508717-de8e2138-d34a-4808-b2a3-94edcc60bccc.png)
 
I have try to use a dialog inside the input field 
https://codepen.io/akashramtohul4/pen/KKvONEP?editors=1011
Can some advice how can i make it look like the image 

Thanks

**Comment by metalsadman** [comment]
use QMenu https://quasar.dev/vue-components/menu#positioning as for the arrow, ~~try googling for css on how to do it~~.

took a shot on doing it here https://codepen.io/metalsadman/pen/GRvVWvq thanks @Evertvdw for the css fix.

**Comment by Evertvdw** [comment]
I myself use a QTooltip for this, which shows on hover, here is what I use:
https://gist.github.com/Evertvdw/3ffd2d4d0159569d67744ef64b9b980f
